third_party/move/move-vm/runtime/src/loader/type_loader.rs

```diff
@@ -4,14 +4,14 @@
 use move_binary_format::{
     binary_views::BinaryIndexedView, errors::PartialVMResult, file_format::SignatureToken,
 };
-use move_vm_types::loaded_data::runtime_types::{AbilityInfo, StructIdentifier, Type};
-use std::sync::Arc;
+use move_vm_types::loaded_data::runtime_types::{AbilityInfo, StructNameIndex, Type};
+use triomphe::Arc;
```

Comment:
I think it might be a good idea to `use triomphe::Arc as TriompheArc`, or something like that - to make it easier for the code readers to distinguish between regular Arc and TriompheArc.